libs/src/sdk/api/chats/ChatsApi.ts

```diff
@@ -18,20 +17,34 @@ import type {
   RPCPayloadRequest,
   ValidatedChatPermissions
 } from './serverTypes'
-import type {
+import {
   ChatBlockRequest,
+  ChatBlockRequestSchema,
   ChatCreateRequest,
+  ChatCreateRequestSchema,
   ChatDeleteRequest,
+  ChatDeleteRequestSchema,
   ChatEvents,
   ChatGetAllRequest,
+  ChatGetAllRequestSchema,
   ChatGetMessagesRequest,
+  ChatGetMessagesRequestSchema,
   ChatGetPermissionRequest,
+  ChatGetPermissionRequestSchema,
   ChatGetRequest,
+  ChatGetRequestSchema,
   ChatInviteRequest,
+  ChatInviteRequestSchema,
   ChatMessageRequest,
+  ChatMessageRequestSchema,
   ChatPermitRequest,
+  ChatPermitRequestSchema,
   ChatReactRequest,
+  ChatReactRequestSchema,
   ChatReadRequest,
+  ChatReadRequestSchema,
+  ChatUnfurlRequest,
+  ChatUnfurlRequestSchema,
   TypedCommsResponse,
   UnfurlResponse
```

Comment:
Kinda feels like we should have a `schema` file that you can `import * as chatSchemas from `./schemas.ts` to keep this a little more tame?

wdyt @sliptype / @nicoback2